src/util/PopulationsMap.ts

```diff
@@ -54,3 +55,89 @@ export function getFhirMeasurePopulationCode(population: string) {
     }
   }
 }
+
+export function triggerPopChanges(
+  groupPopulations: GroupPopulation[],
+  changedId
+) {
+  let returnPop: GroupPopulation[] = [];
+  const expectedValue = groupPopulations[0]?.populationValues.filter(
+    (population) => population.name === changedId
+  )[0]?.expected;
+  returnPop.push(groupPopulations[0]);
+  let myMap = {};
+
+  //iterate through
+  groupPopulations[0].populationValues.forEach((value: PopulationValue) => {
+    myMap[value.name] = value;
+  });
+
+  if (groupPopulations[0].scoring === "Proportion") {
+    //denominator
+    if (changedId === "denominator") {
+      if (expectedValue === true) {
+        myMap[MeasurePopulation.INITIAL_POPULATION].expected = true;
+      }
+
+      if (expectedValue === false) {
+        myMap[MeasurePopulation.NUMERATOR].expected = false;
+        myMap[MeasurePopulation.DENOMINATOR_EXCLUSION] !== undefined &&
+          (myMap[MeasurePopulation.DENOMINATOR_EXCLUSION].expected = false);
```

Comment:
Would the safe navigation work the same way here?
```suggestion
          (myMap[MeasurePopulation.DENOMINATOR_EXCLUSION]?.expected = false);
```

Reply:
No adding "?" to check if it present or not doesn't work here